parity/cli/mod.rs

```diff
@@ -646,7 +646,7 @@ usage! {
 
 			FLAG flag_tx_queue_no_unfamiliar_locals: (bool) = false, or |c: &Config| c.mining.as_ref()?.tx_queue_no_unfamiliar_locals.clone(),
 			"--tx-queue-no-unfamiliar-locals",
-			"Transactions received via local means (RPC, WS, etc) will be treated as external if the sending account is unknown.",
+			"Transactions received via local means (HTTP JSON-RPC, WS JSON-RPC, etc) will be treated as external if the sending account is unknown.",
```

Comment:
Here: WebSockets or WS?

Reply:
:+1: I left it as "ws" was not part of the flag, but you're right, I'll change it to WebSockets.